code/game/objects/items/mountable_frames/trophymount.dm

```diff
@@ -0,0 +1,111 @@
+/obj/item/mounted/frame/trophy_mount
+	name = "trophy mount"
+	desc = "Used to hang your old hunting trophies. "
+	icon = 'icons/obj/objects.dmi'
+	icon_state = "wallmount"
+	anchored = 0
+	var/initial_name = "wooden wall mount"
```

Comment:
it's kinda weird that the name it reverts to is different than the name when you first build it....and again, you can just use `initial(name)`